repeated-integration-tests.bash

```diff
@@ -64,12 +64,9 @@ while [ "${runcount}" -lt "${runs}" ]; do
     timestamp="$(date -Iminutes)"
     client_logfile="$DIR/target/${POD_NAME}-${timestamp}.client.log"
     server_logfile="$DIR/target/${POD_NAME}-${timestamp}.server.log"
-    mkdir -p "$(dirname $client_logfile)"
-    mkdir -p "$(dirname $server_logfile)"
-    >"${client_logfile}"
```

Comment:
I'm wondering what these redirections were for before? Shellcheck said they were redirecting to nowhere and removing them didn't seem to change any functionality, but I'm not sure if there was some subtle change.

Reply:
It redirects an empty stream into the named file. This has the effect of creating the file if it doesn't already exist, or if it did exist then it truncates its contents to the empty stream's contents.

```bash
$ cat foo
cat: foo: No such file or directory
$ >foo
$ cat foo
$ echo hello > foo
$ cat foo
hello
$ >foo
$ cat foo
$ rm foo
$ >foo
$ cat foo
```

Reply:
Should I revert it?

Reply:
I think `mkdir` lines would only do anything if you tried to `bash repeated-integration-tests.bash` without a `$cryostat_dir/target` directory existing, ie you haven't run a build before. The empty redirection would be very unlikely to ever do anything because the file it targets has the current timestamp in its filename, and the later bits that write logs to that file should be fine with creating the file and appending to it without it already existing.

So I guess it's okay to remove them, but it seems harmless to keep these checks even if shellcheck thinks they're ineffective.